app/templates/main/date-filters.html

```diff
@@ -57,7 +69,7 @@
                            id="date_to_day"
                            name="date_to_day"
                            type="text"
-                           value="{% if filters['date_range'] %}{% if filters['date_range']['date_to'] %}{{ filters['date_range']['date_to'][0:2] }}{% endif %}{% endif %}"
+                           value="{% if filters['date_range'] %}{% if filters['date_range']['date_to'] %}{{ filters['date_range']['date_to'][0:2].replace('#', '') }}{% endif %}{% endif %}"
```

Comment:
whats this replacement of hashes for?

Reply:
as the code returns '#' for blank values to set in correct format, replace require to change from '#' to '' value